tests/e2e/test_api.py

```diff
@@ -981,6 +981,95 @@ def _perform_query(client, conversation_id, query):
     print(vars(response))
 
 
+@retry(max_attempts=3, wait_between_runs=10)
+def test_valid_question_with_empty_attachment_list() -> None:
+    """Check the REST API /v1/query with POST HTTP method using empty attachment list."""
+    endpoint = "/v1/query"
+
+    with metrics_utils.RestAPICallCounterChecker(
+        metrics_client, endpoint, status_code=requests.codes.ok
+    ):
+        response = client.post(
+            endpoint,
+            json={
+                "conversation_id": "",
+                "query": "what is kubernetes?",
+                "attachments": [],
+            },
+            timeout=LLM_REST_API_TIMEOUT,
+        )
+
+        # HTTP OK should be returned
+        assert response.status_code == requests.codes.ok
+
+        check_content_type(response, "application/json")
+
+
+@retry(max_attempts=3, wait_between_runs=10)
+def test_valid_question_with_one_attachment() -> None:
+    """Check the REST API /v1/query with POST HTTP method using one attachment."""
+    endpoint = "/v1/query"
+
+    with metrics_utils.RestAPICallCounterChecker(
+        metrics_client, endpoint, status_code=requests.codes.ok
+    ):
+        response = client.post(
+            endpoint,
+            json={
+                "conversation_id": "",
+                "query": "what is kubernetes?",
+                "attachments": [
+                    {
+                        "attachment_type": "log",
+                        "content_type": "text/plain",
+                        "content": "this is attachment",
```

Comment:
i know the implementation isn't in place yet but ultimately we're going to need a test that actually ensures the attachment content gets used in the prompt to the llm, so when you add the implementation part that actually adds the attachment content to the prompt, please keep that in mind (probably best to just update this test at that point, rather than adding another one)

Reply:
yes, it was/is my plan